### ISSUE TITLE
Please add support for the latest AngularJS versions (1.0.3)

### ISSUE THREAD

**Issue by pkozlowski-opensource** [jsfiddle/jsfiddle-issues#238]
Very similar to the https://github.com/jsfiddle/jsfiddle-docs-alpha/issues/221

A new AngularJS  versions was released recently: 1.0.3 (more info here: http://blog.angularjs.org/search/label/release) so it would be great if you could add it to the AngularJS section in the frameworks tab.

File to be included (hosted on Goole's CDN):
- For `1.0.3`: http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular.js

There is one more thing: the full AngularJS distribution contains several files (see below) so it would be really awesome if we could have something like AngularJS `1.0.3` _and_ `AngularJS 1.0.3 (full)`. In the full version files to be included would be (order matters):
- http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular.js
- http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular-sanitize.js
- http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular-resource.js

Alternatively, if you don't mind referencing those 3 files all the time we could have only one entry with the 3 files listed above.
